code/game/machinery/embedded_controller/mapping_helpers.dm

```diff
@@ -0,0 +1,176 @@
+/*
+
+Note that these have to be in the same /area that the controller is in for them to function.
+You still need to set the controller's "id_tag" to something unique.
+Any frequency works, it's self-setting, but it seems like people have decided 1380 for airlocks so maybe set that on the controller too.
+
+*/
+
+/obj/effect/map_helper/airlock
+	name = "use a subtype!"
+	icon = 'icons/misc/map_helpers.dmi'
+	plane = 20 //I dunno just high.
+	alpha = 170
+
+	//The controller we're wanting our device to use
+	var/obj/machinery/embedded_controller/radio/my_controller
+	var/my_controller_type = /obj/machinery/embedded_controller/radio/airlock
+	//The device we're setting up
+	var/my_device
+	var/my_device_type
+	//Most things have a radio tag of some sort that needs adjusting
+	var/tag_addon
+
+/obj/effect/map_helper/airlock/Initialize()
+	..()
+	my_controller = get_controller(get_area(src))
```

Comment:
Can these be mapped in with particular values (As airlocks are mapped right now [A process of which I am generally unaware, so this may be nonsense]) such that if someone were to, for some reason, try and stuff five different airlocks right next to each other, so that the helpers in one aren't seeing the controller in the other?

Reply:
They find the nearest one a few lines below, so you can have several in the same area without worrying. Tether has like 8 in one area at the dock.